### PR TITLE
Feature/fix doc links

### PR DESCRIPTION
**Pull request by v-anshie**
This PR removes the links to outdated apple documentation in each resource documentation, and adds links to each resource that is being defined in the documentation.

### CODE REVIEW

File: documentation/resource_ard.md [hunk 1]
```diff
@@ -23,15 +20,15 @@ end
...
+- `:configure` configures the agent using the `kickstart` default commandline arguments.
 
 The default `:configure` action is equivalent to the following
 **System Preferences > Sharing** settings:
 
 ![Sharing Preferences](sharing_preferences.png)
 
 The full syntax for all of the properties that are available to the **ard**
-resource is:
+resource are:
```
Comment (jazaval) [inline documentation/resource_ard.md]: This should be `is` since we're referring to the singular `syntax`, not the plural `properties`.

Comment (v-anshie) [reply documentation/resource_ard.md]: You are listing all of the properties though. So you are also referring to the plural properties?

File: documentation/resource_ard.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
 ard
 ===
 
-Use the **ard** resource to manage the "Remote Management" settings, found in System
-Preferences > Sharing > Remote Management. Under the hood, an **ard** resource
+Use the [**ard**](https://github.com/Microsoft/macos-cookbook/blob/master/resources/ard.rb) resource to manage the "Remote Management" settings, found in System
```
[the remark this answers is not in the document]
Comment (v-anshie) [reply documentation/resource_ard.md]: Makes sense, I will do this for all of the resources that I made this change too. Thanks!

File: documentation/resource_ard.md (hunk 1 above)
Comment (v-anshie) [reply documentation/resource_ard.md]: Changed these back, thanks for the clarification.

**Pull request merged**